### PR TITLE
chore: initial docker configuration

### PR DESCRIPTION
**Pull request by ronaldo-mendoza-jalafund**
#### Resolved task/US: ([https://app.clickup.com/t/868b5ma6p]())
* * * *

## What's in this pull request?
Write a Dockerfile to build an image of the UI. 
Write a dockerfile-compose.yaml file to run the UI.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules
+.git
+*.zip
+*.log
+dist
+build
```
Comment (JosuePrado) [inline .dockerignore]: missing end line

File: Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+RUN npm run build
+
+FROM nginx:alpine
+
+COPY --from=build /app/dist /usr/share/nginx/html
+
+EXPOSE 80
+
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (JosuePrado) [inline Dockerfile.dev]: missing end line

File: Dockerfile.prod [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+RUN npm run build
+
+FROM nginx:alpine
+
+COPY --from=build /app/dist /usr/share/nginx/html
+
+EXPOSE 82
+
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (JosuePrado) [inline Dockerfile.prod]: missing end line

**Comment by JosuePrado** [comment]
After running the production docker compose, I can't see the application running on the designated port for that container, can you change it or explain me how to see it?

**Review by ronaldo-mendoza-jalafund — COMMENTED** [review]
In dockerfiles it shows that it is exporting on port 80 in Dev and on port 82 in

**Review by JosuePrado — APPROVED** [review]

**Review by GabyGV-GGV — APPROVED** [review]
Good Job!

**Pull request merged**